source/_config.yml

```diff
@@ -318,6 +318,14 @@ documents:
       puppetdb: "4.3"
     hide: true
 
+  /puppet/6.4:
+    doc: puppet
+    version: "6.4"
+    nav: ./_puppet_toc.html
+    my_versions:
+      facter: "3.13"
```

Comment:
This version gets Facter 3.13, but these docs are now integrated into Puppet docs in ezd. So I think maybe I can/should remove this line?